pkg/virt-controller/watch/vmi/vmi_test.go

```diff
@@ -3609,6 +3611,60 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			expectMatchingPodCreation(vmi)
 		})
 	})
+
+	Context("Event handling", func() {
+		indexVirtualMachine := func(vmi *virtv1.VirtualMachineInstance) {
+			Expect(controller.vmiIndexer.Add(vmi)).To(Succeed())
+			_, err := virtClientset.KubevirtV1().VirtualMachineInstances(vmi.Namespace).Create(context.Background(), vmi, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+		}
+
+		pvcWith := func(requestSize, capacity, rv string) *k8sv1.PersistentVolumeClaim {
+			pvc := newPvc("default", "testpvc")
+			pvc.Spec.Resources.Requests = k8sv1.ResourceList{
+				k8sv1.ResourceStorage: resource.MustParse(requestSize),
+			}
+			pvc.Status.Capacity = k8sv1.ResourceList{
+				k8sv1.ResourceStorage: resource.MustParse(capacity),
+			}
+			pvc.ObjectMeta.ResourceVersion = rv
+			return pvc
+		}
+
+		DescribeTable("updatePVC", func(newReq, newCap, newRV string, expectEnqueue bool) {
+			vmi := libvmi.New(
+				libvmi.WithName("testvmi"),
+				libvmi.WithNamespace("default"),
+				libvmi.WithDataVolume("myvolume", "testpvc"),
+			)
+			indexVirtualMachine(vmi)
```

Comment:
indexVirtualMachine > indexVirtualMachineInstance ?
or just addVMI should be enough :)

Reply:
ack